### ISSUE TITLE
Current configuration is confusing/redundant

### ISSUE THREAD

**Issue by mieubrisse** [kurtosis-tech/ethereum-package#282]
As an outsider/user, I found the current package configuration hard to reason about due to the organic way configs were added, resulting in multiple configuration values controlling the same thing. Some examples:

- it's not clear to me why `launch_additional_services` and `additional_services` are separate, and why launch_additional_services: False can't simply  be specified by an empty array
- it was confusing to me that tx_spammer_params, goomy_blob_params, and explorer_version are separate top-level params (I would have expected them to be tied to the service, if and only if I set that service. When it's top-level, I assume it's always relevant)
- with MEV, I'm not sure if this is a sidecar per client, or a completely different thing entirely that gets instantiated. from its position as a top-level config object, I would expect it to be completely independent from the participants
- `snooper_enabled` is configurable in two places - both at the client-pair level, and at the global level. It's not clear to me how these interact: is the global level the master control? is the client pair level just an override to the global level? if it's just an override, does the fact that it defaults to false then override the global?

**Comment by barnabasbusa** [comment]
* `launch_additional_services` can be deprecated indeed, and `additional_services` empty array should represent that no additional tooling is spun up.
* `explorer_version` can be removed. `*_params` are service related parameters that you might want to change per enclave. 
* mev spins up a set of tooling, mev-relay, mev-dbs, housekeeper, as well as a block builder, which is currently a LH/Geth pair. When MEV is defined, these tools are always spun up. So if you define 4 participants by default + mev, then you will get 5 participants, last one always being a geth/lighthouse pair that behaves as a builder. It is completely independent from the participant list, other than the args that are run for the geth/lighthouse, as the arguments are concatenated with the builder arguments. Mev-boost is an another application that is run when mev is enabled, and this allows the CLs to be configured with their mev functionality enabled, and configures all blocks to be built by the mev builder instead of doing local block building. 
* snooper is a tool that sits between the CL and EL. I have implemented it as a client specific flag or a global flag, as you might want to enable the snooper for only a specific client or globally for all clients. It defaults to false, and by default there is no snooper. The only reason I did client specific snooper also, is because there might be a case where you run 50 nodes, and you don't want to run 50 snoopers but only want to run snoopers for the first x clients. But day to day, on a small testnet its very very useful to have the snooper image running for each pair, and hence we have the global flag.
